src/Command/SF1500/SF1500LookupCommand.php

```diff
@@ -0,0 +1,267 @@
+use ItkDev\Serviceplatformen\Service\SF1500\SF1500XMLBuilder;
+use ItkDev\Serviceplatformen\Service\SF1514\SF1514;
+use ItkDev\Serviceplatformen\Service\SoapClient;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+class SF1500LookupCommand extends Command
+{
+    const SF1500_TEST_MANAGER_FUNKTIONSTYPE_ID = '46c73630-f7ad-4000-9624-c06131cde671';
+    const SF1500_PROD_MANAGER_FUNKTIONSTYPE_ID = 'to_be_found';
+
+    protected static $defaultName = 'serviceplatformen:sf1500:lookup';
+
+    private array $inputOptions = [];
+
+    protected function configure()
+    {
+        $inputOptions = [
+            new InputOption('manager', null, InputOption::VALUE_NONE, 'get information on manager for provided user-id'),
+            new InputOption('production', null, InputOption::VALUE_NONE, 'production mode'),
+            new InputOption('user-id', null, InputOption::VALUE_REQUIRED, 'user id'),
+            new InputOption('certificate', null, InputOption::VALUE_REQUIRED, 'path to certificate'),
+            new InputOption('passphrase', null, InputOption::VALUE_OPTIONAL, 'passphrase for certificate', ''),
+            new InputOption('authority-cvr', null, InputOption::VALUE_REQUIRED, 'authority cvr', '55133018'),
+            new InputOption('sts-applies-to', null, InputOption::VALUE_REQUIRED, 'service SAML token should apply to'),
+        ];
+        $this->setDefinition(new InputDefinition($inputOptions));
+
+        $this->setDescription('Helper script for testing “SF1500: Organisation”.');
+
+        $help = <<<HELP
+
+user-id:
+    a user id, e.g.
+    'ffdb7559-2ad3-4662-9fd4-d69859939b66'
+
+certificate:
+    The certificate option can be a file path to a PKCS #12 certificate or a url query string with options for getting the certificate from an Azure Key Vault, e.g. (newlines added for readability and proper url encoding skipped)
+
+    tenant_id=f1d3b517-acac-4902-ae36-aaa32ed814dd
+    &client_id=7d711706-4325-47a6-a8c4-b373615c2b75
+    &client_secret=6e8551bf-2bb0-46bc-b449-647720120495
+    &name=organisation
+    &secret=test
+    &version=bbcbd812-0d7c-420b-9b93-7318e3769578
+    
+    If using path option make sure the path is relative to project root to the certificate file, e.g.
+    '/app/src/Command/SF1500/certificate.p12'
+ 
+passphrase:
+    the passphrase for p12 certificate, i.e.
+    'XYZ'
+    defaults to the empty string if not provided
+
+sts-applies-to:
+    the service SAML token should grant access to, i.e. SF1500 Organisation
+    'http://stoettesystemerne.dk/service/organisation/3'
```

Comment:
Can this apply to anything else than http://stoettesystemerne.dk/service/organisation/3?

Reply:
Not as long as the service isn't updated to a new version. I suppose should hard code this then.

Reply:
Which other values can be used (in SF1514)? Does something similar to https://github.com/itk-dev/serviceplatformen/blob/develop/src/Service/SF1601/SF1601.php#L77-L81 make sense in SF1514?

Reply:
Actually, it is the same for both test and production, it's a way of telling SF1514 which service the SAML-token should grant access to.

As we don't have other services that use the same security methodology hard coding it seems fine.